.github/workflows/test.yml

```diff
@@ -236,8 +236,72 @@ jobs:
           E2E_CI_TIMEOUT_SEC: '600'
           E2E_KATHY_MESSAGES: '20'
 
-      - name: cli e2e tests
-        run: ./typescript/cli/ci-test.sh
+  cli-e2e:
+    runs-on: larger-runner
+    needs: [yarn-build]
+    strategy:
+      matrix:
+        include:
+          - hook: false
+          - hook: true
```

Comment:
imo it would be nice to configure this with a hook config file name
as it stands its hard to know what the hook: true means

Reply:
fixed